autorandr.py

```diff
@@ -1192,15 +1189,21 @@ def exec_scripts(profile_path, script_name, meta_information=None):
                 if check_name not in ran_scripts:
                     script = os.path.join(script_folder, file_name)
                     if os.access(script, os.X_OK | os.F_OK):
-                        try:
-                            all_ok &= subprocess.call(script, env=env) != 0
-                        except:
-                            raise AutorandrException("Failed to execute user command: %s" % (script,))
+                        all_ok &= exec_script(script, env) != 0
                         ran_scripts.add(check_name)
 
     return all_ok
 
 
+def exec_script(script, env):
```

Comment:
Please add a docstring explaining what this function does. A one-liner is fine. Also, how about `exec_one_script` so there won't be any confusion about which of the functions is internal?

Reply:
Done.